build.sbt

```diff
@@ -216,7 +218,7 @@ lazy val libManagedSettings = Seq(
       schemas.map { schema =>
         val out = outdir / "org" / "apache" / "daffodil" / "xsd" / schema.getName
         IO.copyFile(schema, out)
-        stream.log.info("Generated %s".format(out))
+        stream.log.info("generated %s".format(out))
```

Comment:
This was done just to make output more consistent with sbt output. They've starting using lowercase in lots of places.